sqlitecluster/SQLiteNode.cpp

```diff
@@ -1278,7 +1278,19 @@ void SQLiteNode::_onMESSAGE(SQLitePeer* peer, const SData& message) {
             peer->commandAddress = message["commandAddress"];
         }
         peer->setCommit(message.calcU64("CommitCount"), message["Hash"]);
-
+        
+        // We check the commit difference with 12,500 commits behind because that 
+        // represents ~30s of commits. If we're behind, let's close the command port
+        // so we can catch up with the cluster before processing new commands.
+        const int64_t currentCommitDifference = message.calcU64("NewCount") - getCommitCount();
```

Comment:
Not all messages have `newCount` but all messages have `CommitCount`, which we just checked a few lines above. But also we don't care what the peer's commit count is if it isn't leading.

Given the previous line, we can do:
```
if (peer == leadPeer && peer->commitCount > getCommitCount() + 12'500) 
```

And we don't need to parse the number from the message again.